test/integration/targets/get_certificate/tasks/main.yml

```diff
@@ -1,5 +1,28 @@
 - block:
 
-    - include_tasks: ../tests/validate.yml
+  - include_tasks: ../tests/validate.yml
+    vars:
+      select_crypto_backend: pyopenssl
 
   when: pyopenssl_version.stdout is version('0.15', '>=')
+
+- name: Remove output directory
+  file:
+    path: "{{ output_dir }}"
+    state: absent
+
+- name: Re-create output directory
+  file:
+    path: "{{ output_dir }}"
+    state: directory
+
+- block:
+
+  - include_tasks: ../tests/validate.yml
+    vars:
+      select_crypto_backend: cryptography
+
```

Comment:
Should a test be included for if select_crypto_backend is not set at all, for backwards compatibility?

Reply:
Good point. I'll add something later.